### PR TITLE
add geologicalProject

### PR DESCRIPTION
**Pull request by OsamaNajjar**
dear @SameerAlHarbi 

please reviw new code for geological Projects

### CODE REVIEW

File: src/routers/delegations.router.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
 const express = require('express');
 const delegationsController = require('../controllers/delegations.controller');
+const geologicalProjectsController = require('../controllers/geological-projects.controller');
 
 const Router = express.Router();
 
 // /delegations/Types?{lang=A} => GET
 Router.get('/types', delegationsController.getDelegationTypes);
 
+// /delegations/geologicalProject => POST body {"projectCode": "000011",
```
Comment (SameerAlHarbi) [inline src/routers/delegations.router.js:10]: // /delegations/projects => POST body {"projectCode": "000011",

File: src/routers/delegations.router.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+//                                              "projectYear": "2022",
+//                                              "CreateProjectEmployeeId": "917",
+//                                              "projectManagerId": "1143",
+//                                              "projectTitle": "Test Add Project from Node.js",
+//                                              "projectNote": "Test",
+//                                              "isActive": "true"}
+Router.post('/geologicalProject', geologicalProjectsController.addOrEditGeologicalProject);
```
Comment (SameerAlHarbi) [inline src/routers/delegations.router.js:17]: Router.post('/projects', geologicalProjectsController.addOrEditGeologicalProject);

File: src/routers/delegations.router.js [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
 const delegationsController = require('../controllers/delegations.controller');
+const geologicalProjectsController = require('../controllers/geological-projects.controller');
 
 const Router = express.Router();
 
 // /delegations/Types?{lang=A} => GET
 Router.get('/types', delegationsController.getDelegationTypes);
 
+// /delegations/geologicalProject => POST body {"projectCode": "000011",
+//                                              "projectYear": "2022",
```
Comment (SameerAlHarbi) [inline src/routers/delegations.router.js:11]: // "year": "2022",

File: src/controllers/geological-projects.controller.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const geologicalProjectsManager = require('../managers/geological-projects.manager');
```
Comment (SameerAlHarbi) [inline src/controllers/geological-projects.controller.js:1]: const projectsManager = require('../managers/projects.manager');

File: src/controllers/geological-projects.controller.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+exports.addOrEditGeologicalProject = async (req, res, next) => {
```
Comment (SameerAlHarbi) [inline src/controllers/geological-projects.controller.js:3]: exports.saveProject = async (req, res, next) => {

File: src/controllers/geological-projects.controller.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    
+    const geologicalProject = req.body;
```
Comment (SameerAlHarbi) [inline src/controllers/geological-projects.controller.js:5]: const project = req.body;

**Review by SameerAlHarbi — CHANGES_REQUESTED** [review]
Please review this changes

**Comment by SameerAlHarbi** [comment]
Please pull this branch before change

**Comment by OsamaNajjar** [comment]
Please review new code for Projects

**Pull request merged**